content/source/docs/enterprise/api/ptfe_settings.html.md

```diff
@@ -0,0 +1,491 @@
+---
+layout: enterprise2
+page_title: "Private Terraform Settings - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-private-terraform-settings"
```

Comment:
When file moves, slug should change to `docs-enterprise2-api-admin-settings` so it lights up in the appropriate nav sidebar section.